src/components/List.jsx

```diff
@@ -31,6 +37,11 @@ const List = ({ id, list, booksInList, refetchLists, refetchBooks }) => {
     }
   }
 
+  function getListName(e) {
+    const elementClickedId = e.currentTarget.parentElement.parentElement.parentElement.parentElement.id;
+    setList(elementClickedId)
```

Comment:
I'm not crazy about this chain, we already have access to the id in the list prop